gunicorn.conf.py

```diff
@@ -1,14 +1,24 @@
 import os
 DEPLOYMENT_MODE = os.environ.get('DEPLOYMENT_MODE')
 
+worker_class = 'gevent'
+workers = 4
+# to stay under heroku limit of 20 connections
+worker_connections = 4
 
 pythonpath = "/app/joplin"
 reload = True
 #  see https://docs.gunicorn.org/en/stable/settings.html#max-requests
-max_requests = 100
+max_requests = 500
```

Comment:
I upped this as it might help prevent timeouts, and the individual works _seem_ to be able to handle more simultaneous requests under this model than previously, and the startup cost of spinning up a new workers just cause it's handled 100 requests may be slowing us down (fetching route data, for example, seems to corelate to fetching around 300 requests as a single pass?)